### PR TITLE
Add event_creator worker

### PR DESCRIPTION
**Pull request by erikjohnston**

### CODE REVIEW

File: tests/61push/03_unread_count.pl [hunk 1]
```diff
@@ -25,6 +25,8 @@
          my ( $event_id ) = @_;
 
          matrix_advance_room_receipt_synced( $user1, $room_id, "m.read" => $event_id );
+      })->then( sub {
+         delay( 0.5 )
```
Comment (richvdh) [inline tests/61push/03_unread_count.pl]: ugh

Comment (erikjohnston) [reply tests/61push/03_unread_count.pl]: Oh, I had forgotten about that :/

Comment (erikjohnston) [reply tests/61push/03_unread_count.pl]: In fact, I think I'm being a crank and uploaded something I shouldn't have...

**Comment by erikjohnston** [comment]
(I removed the offending delay stuff, apologies for crankness)

**Pull request merged**